.github/workflows/te-review.yml

```diff
@@ -0,0 +1,280 @@
+  K8S_PROBE_PERIOD: 20
+  EMPLOYER_URL: https://te-empl-${{ github.event.pull_request.number }}.${{ secrets.BASE_DOMAIN_STAGING }}
+  YOUTH_URL: https://te-yout-${{ github.event.pull_request.number }}.${{ secrets.BASE_DOMAIN_STAGING }}
+  ADMIN_URL: https://te-admn-${{ github.event.pull_request.number }}.${{ secrets.BASE_DOMAIN_STAGING }}
+  NEXT_PUBLIC_BACKEND_URL: https://te-bknd-${{ github.event.pull_request.number }}.${{ secrets.BASE_DOMAIN_STAGING }}
+  MOCK_FLAG: 1
+  NEXT_PUBLIC_SENTRY_DSN: ${{ secrets.NEXT_PUBLIC_SENTRY_DSN }}
+  NEXT_PUBLIC_SENTRY_ENVIRONMENT: ${{ secrets.NEXT_PUBLIC_SENTRY_ENVIRONMENT }}
+  CORS_ALLOW_ALL_ORIGINS: 1
+jobs:
+  build:
+    if: ${{ github.actor != 'dependabot[bot]' }}
+    strategy:
+      fail-fast: false
+      matrix:
+        service: [ 'te-bknd', 'te-empl', 'te-yout', 'te-admn' ]
+        include:
+          - service: te-bknd
+            context: ./backend
+            dockerfile: ./backend/docker/tet.Dockerfile
+            port: 8000
+          - service: te-empl
+            context: ./frontend
+            dockerfile: ./frontend/tet/employer/Dockerfile
+            port: 3000
+          - service: te-yout
+            context: ./frontend
+            dockerfile: ./frontend/tet/youth/Dockerfile
+            port: 3000
+          - service: te-admn
+            context: ./frontend
+            dockerfile: ./frontend/tet/admin/Dockerfile
+            port: 3000
+    runs-on: ubuntu-latest
+    name: Build
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build ${{ matrix.service }}
+        if: github.event_name == 'pull_request' || github.event.inputs.build_required == 'true'
+        uses: andersinno/kolga-build-action@v2
+        env:
+          DOCKER_BUILD_ARG_NEXT_PUBLIC_BACKEND_URL: ${{ env.NEXT_PUBLIC_BACKEND_URL }}
+          DOCKER_BUILD_ARG_NEXT_PUBLIC_SENTRY_DSN: ${{ secrets.NEXT_PUBLIC_SENTRY_DSN }}
+          DOCKER_BUILD_ARG_NEXT_PUBLIC_SENTRY_ENVIRONMENT: ${{ secrets.NEXT_PUBLIC_SENTRY_ENVIRONMENT }}
+          DOCKER_BUILD_SOURCE: ${{ matrix.dockerfile }}
+          DOCKER_BUILD_CONTEXT: ${{ matrix.context }}
+          DOCKER_IMAGE_NAME: ${{ matrix.service }}
+          SERVICE_PORT: ${{ matrix.port }}
+          BUILDKIT_CACHE_DISABLE: true
+  review:
+    strategy:
+      fail-fast: false
+      matrix:
+        service: [ 'te-bknd', 'te-empl', 'te-yout', 'te-admn' ]
+        include:
+          - service: te-bknd
+            context: ./backend
+            dockerfile: ./backend/docker/tet.Dockerfile
+            port: 8000
+          - service: te-empl
+            context: ./frontend
+            dockerfile: ./frontend/tet/employer/Dockerfile
+            port: 3000
+          - service: te-yout
+            context: ./frontend
+            dockerfile: ./frontend/tet/youth/Dockerfile
+            port: 3000
+          - service: te-admn
+            context: ./frontend
+            dockerfile: ./frontend/tet/admin/Dockerfile
+            port: 3000
```

Comment:
`database: false/true` are missing from these. See [kesäseteli](https://github.com/City-of-Helsinki/yjdh/blob/develop/.github/workflows/ks-review.yml#L88:L103).